### PR TITLE
PAYARA-3540: Use no-fork javadoc invocation and package the jar manually

### PR DESCRIPTION
**Pull request by pdudits**
Alternative implementation to #3826

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: pom.xml [hunk 1]
```diff
@@ -194,18 +195,42 @@
...
+                                        directly with jar plugin
+                                    -->
+                                    <!--<goal>aggregate-jar</goal>-->
                                 </goals>
                                 <phase>pre-site</phase>
                                 <configuration>
                                     <!-- Specific configuration for the aggregate report -->
                                     <finalName>payara-5.192-SNAPSHOT</finalName>
+                                    <minmemory>512m</minmemory>
+                                    <maxmemory>2g</maxmemory>
```
Comment (Pandrex247) [inline pom.xml]: Does this override the settings from -Xmx & -Xms?

**Comment by Pandrex247** [comment]
Aside from my one question above - fixes the issue

File: pom.xml (hunk 1 above)
Comment (pdudits) [reply pom.xml]: I believe so. The javadoc process got more convenient heap

**Pull request merged**